doc/source/PreConfiguredSites.rst

```diff
@@ -64,8 +64,10 @@ Pre-configured sites (tier 1)
 |                     | Jet                   | GCC, Intel         | ``/contrib/spack-stack``                               | EPIC / NOAA-EMC |
 +---------------------+-----------------------+--------------------+--------------------------------------------------------+-----------------+
 |                     | Narwhal               | GCC, Intel, oneAPI | ``/p/app/projects/NEPTUNE/spack-stack/``               | NRL             |
-| U.S. Navy (HPCMP)   +-----------------------+--------------------+--------------------------------------------------------+-----------------+
-|                     | Nautilus              | GCC, Intel, oneAPI | ``/p/app/projects/NEPTUNE/spack-stack/``               | NRL             |
+|                     +-----------------------+--------------------+--------------------------------------------------------+-----------------+
+| U.S. Navy (HPCMP)   | Nautilus              | GCC, Intel, oneAPI | ``/p/app/projects/NEPTUNE/spack-stack/``               | NRL             |
+|                     +-----------------------+--------------------+--------------------------------------------------------+-----------------+
+|                     | Blueback (earlyaccess)| GCC, oneAPI        | (experimental only, no project directories yet         | NRL             |
 +---------------------+-----------------------+--------------------+--------------------------------------------------------+-----------------+
 | Univ. of Wisconsin  | S4                    | Intel              | ``/data/prod/jedi/spack-stack/``                       | JCSDA           |
```

Comment:
We had a chat this afternoon and Innocent will be maintaining the 1.6 site config for S4 and doing the 1.9 site install. So you can update this and remove JCDDA.

Per your other question in the PR description, you can leave Discover's maintainers as is.

Reply:
Ok, thanks. I'll make the maintainer SSEC for S4. And clean up the `MaintainersSection.rst` file before I merge.